packages/nodes-base/nodes/TheHive/TheHive.node.ts

```diff
@@ -253,7 +253,7 @@ export class TheHive implements INodeType {
 						value: 'executeResponder',
 						description: 'Execute a responder on the specified task',
 					},
-					{ name: 'Get All', value: 'getAll', description: 'Get all asks of a specific case' },
+					{ name: 'Get Many', value: 'getAll', description: 'Get all asks of a specific case' },
```

Comment:
do we want to change it in load methods?

Reply:
The only reason The Hive is using load options is to populate them based on API version, likely because versioning did not exist when this node was created. Same use case otherwise, so keeping it.